Octokit.Tests.Integration/Clients/RepositoryCommitsClientTests.cs

```diff
@@ -158,6 +166,17 @@ public TestsWithNewRepository()
             Assert.Equal(0, result.BehindBy);
         }
 
+        [IntegrationTest]
+        public async Task GetSha1FromRepository()
+        {
+            await CreateTheWorld();
```

Comment:
I'm not sure how much this will break, but I'd love to get the `Reference` that `CreateTheWorld` returns - it should contain a `Sha`.

So rather than invoking `GetSha1` twice below, you invoke it once and compare that SHA to the one you have here.

Does that make sense?